suite-native/message-system/src/selectors.ts

```diff
@@ -0,0 +1,9 @@
+import { createMemoizedSelector, selectActiveFeatureMessages } from '@suite-common/message-system';
+
+export const selectActiveKillswitchMessages = createMemoizedSelector(
```

Comment:
Why don't you return only one killswitch message directly from here? You never need the whole list, always only the highest priority... so `A.head` can be moved here, I guess?

Reply:
49a065c965